------------------------------------------------------------
revno: 14150
revision-id: squid3@treenet.co.nz-20170331005152-8exm3hsly1v1jk8y
parent: squid3@treenet.co.nz-20170330133122-zcpblbvnuq7mjvq3
committer: Amos Jeffries <squid3@treenet.co.nz>
branch nick: 3.5
timestamp: Fri 2017-03-31 12:51:52 +1200
message:
  Fix variable shadowing after rev.14149
------------------------------------------------------------
# Bazaar merge directive format 2 (Bazaar 0.90)
# revision_id: squid3@treenet.co.nz-20170331005152-8exm3hsly1v1jk8y
# target_branch: http://bzr.squid-cache.org/bzr/squid3/3.5
# testament_sha1: ae1e30fff31cf8b411c62eba344fdc944692aecf
# timestamp: 2017-03-31 01:51:06 +0000
# source_branch: http://bzr.squid-cache.org/bzr/squid3/3.5
# base_revision_id: squid3@treenet.co.nz-20170330133122-\
#   zcpblbvnuq7mjvq3
# 
# Begin patch
=== modified file 'src/client_side.cc'
--- src/client_side.cc	2017-03-30 13:31:22 +0000
+++ src/client_side.cc	2017-03-31 00:51:52 +0000
@@ -4390,7 +4390,6 @@
         } else {
             // in.buf still has the "CONNECT ..." request data, reset it to SSL hello message
             connState->in.buf.append(rbuf.content(), rbuf.contentSize());
-            ClientSocketContext::Pointer context = connState->getCurrentContext();
             ClientHttpRequest *http = context->http;
             tunnelStart(http, &http->out.size, &http->al->http.code, http->al);
         }

